addons/account/migrations/12.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,387 @@
+account      / res.company              / account_onboarding_invoice_layout_state (selection): NEW selection_keys: ['done', 'just_done', 'not_done']
+account      / res.company              / account_onboarding_sale_tax_state (selection): NEW selection_keys: ['done', 'just_done', 'not_done']
+account      / res.company              / account_onboarding_sample_invoice_state (selection): NEW selection_keys: ['done', 'just_done', 'not_done']
+# NOTHING TO DO
+
+account      / res.company              / account_purchase_tax_id (many2one): NEW relation: account.tax
+account      / res.company              / account_sale_tax_id (many2one): NEW relation: account.tax
+??
+
+account      / res.company              / account_setup_bank_data_done (boolean): DEL
+account      / res.company              / account_setup_bank_data_state (selection): NEW selection_keys: ['done', 'just_done', 'not_done']
+account      / res.company              / account_setup_bar_closed (boolean): DEL 
+account      / res.company              / account_setup_coa_done (boolean): DEL 
+account      / res.company              / account_setup_coa_state (selection): NEW selection_keys: ['done', 'just_done', 'not_done']
+account      / res.company              / account_setup_company_data_done (boolean): DEL 
+account      / res.company              / account_setup_fy_data_done (boolean): DEL 
+account      / res.company              / account_setup_fy_data_state (selection): NEW selection_keys: ['done', 'just_done', 'not_done']
+# DONE: post-migration: filled the new *_state fields
+
+account      / res.company              / accounts_code_digits (integer): DEL
+account      / res.company              / incoterm_id (many2one)        : previously in module l10n_be_intrastat
+account      / res.company              / invoice_is_email (boolean)    : NEW 
+account      / res.company              / invoice_is_print (boolean)    : NEW 
+account      / res.company              / invoice_reference_type (selection): NEW selection_keys: function
+account      / res.company              / qr_code (boolean)             : NEW
+account      / res.company              / transfer_account_code_prefix (char): NEW 
+# NOTHING TO DO
+
+---XML records in module 'account'---
+DEL account.financial.report: account.account_financial_report_assets0 (noupdate)
+DEL account.financial.report: account.account_financial_report_balancesheet0 (noupdate)
+DEL account.financial.report: account.account_financial_report_expense0 (noupdate)
+DEL account.financial.report: account.account_financial_report_income0 (noupdate)
+DEL account.financial.report: account.account_financial_report_liability0 (noupdate)
+DEL account.financial.report: account.account_financial_report_liabilitysum0 (noupdate)
+DEL account.financial.report: account.account_financial_report_profitandloss0 (noupdate)
+DEL account.financial.report: account.account_financial_report_profitloss_toreport0 (noupdate)
```

Comment:
You should put a comment saying that this model has dissappeared, so no need to delete these records.